cmd/analyze/flag/create.go

```diff
@@ -65,16 +67,25 @@ func flagCreatedTable(cmd *cobra.Command, listedFlags []models.Flag) error {
 			var flagRequest models.Flag
 			var flagResponse models.Flag
 
+			if analyzedFlag.FlagKey == "" {
+				if !slices.Contains(flagLocationAddedToTable, fmt.Sprintf("%s:%d", r.File, analyzedFlag.LineNumber)) {
```

Comment:
Are you sure it should not be `flagKeyNotDetected` here ? instead of `flagLocationAddedToTable` ?